caddyconfig/caddyfile/formatter_test.go

```diff
@@ -362,6 +362,21 @@ block {
 
 block {
 }
+`,
+		},
+		{
+			description: "keep heredoc as-is",
+			input: `block {
+	heredoc <<HEREDOC
+	Here's more than a space       Here's more than a space
+	<<HEREDOC
```

Comment:
This doesn't look right, a heredoc closing marker does not contain the `<<`.

Reply:
Sorry I forgot that.